### PR TITLE
fixed up content re: projects, quotas, limits

### PR DESCRIPTION
**Pull request by tpoitras**
This content was all interrelated. Based on these docs trello cards:

https://trello.com/c/HWLPY6AR
https://trello.com/c/5J6nXl09
https://trello.com/c/hy4lPYmb
https://trello.com/c/daDSX63E

initial content from these PRs:

https://github.com/openshift/openshift-docs/pull/235
https://github.com/openshift/openshift-docs/pull/286
https://github.com/openshift/openshift-docs/pull/292

### CODE REVIEW

**Comment by tpoitras** [comment]
Tech review please! 

Ping @fabianofranz for #235 and #292 
Ping @derekwaynecarr for #286

**Comment by fabianofranz** [comment]
LGTM, thanks!

File: architecture/core_objects/kubernetes_model.adoc [hunk 1]
```diff
@@ -216,26 +216,23 @@ For more on services, see the https://github.com/GoogleCloudPlatform/kubernetes/
...
 Each user community wants to be able to work in isolation from other communities.
-
 Each user community has its own:
 
-1. resources (pods, services, replication controllers, etc.)
-2. policies (who can or cannot perform actions in their community)
-3. constraints (this community is allowed this much quota, etc.)
+* Resources: pods, services, replication controllers
+* Policies: who can or cannot perform actions in their community
+* Constraints: this community is allowed this much quota
```
Comment (adellape) [inline architecture/core_objects/kubernetes_model.adoc]: Just a suggestion: For these sorts of lists of terms/definitions, I've been using labeled lists with the `[horizontal]` style (http://asciidoctor.org/docs/asciidoc-writers-guide/#labeled-lists and scroll a little).

**Comment by derekwaynecarr** [comment]
LGTM - thanks for improving!

File: architecture/core_objects/openshift_model.adoc [hunk 2]
```diff
@@ -355,7 +355,12 @@ Sample Route:
 ----
 
 == Project
-An OpenShift project allows a community of users to organize and manage their content in isolation from other communities. Each project has its own resources (pods, services, replication controllers, etc.), policies (who can or cannot perform actions), and constraints (this project is allowed this much quota, etc.).  An OpenShift cluster administrator may create an OpenShift project and delegate administrative rights to the project to a member of the user community.
+An OpenShift project allows a community of users to organize and manage their content in isolation from other communities. Each project has its own:
```
Comment (adellape) [inline architecture/core_objects/openshift_model.adoc]: Could make the first "project" in this sentence be a link to the `dev_guide/projects` topic.

File: diff.diff [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+ 
+@@ -175,7 +180,7 @@ service frontend-test (172.30.17.236:5432 -> 8080)
+     #1 deployment waiting on image
+ 
+ To see more information about a service or deployment config, use 'osc describe service <name>' or 'osc describe dc <name>'.
+-You can use 'osc get pods,svc,dc,bc,builds' to see lists of each of the types described above. 
++You can use 'osc get pods,svc,dc,bc,builds' to see lists of each of the types described above.
+ ----
+ 
+ == CLI Object Types
```
Comment (adellape) [inline diff.diff]: I'm guessing this file was just an accidental check-in.

Comment (tpoitras) [inline diff.diff]: Oops, definitely. Removed.

**Comment by tpoitras** [comment]
@adellape -- rolled in your changes, and devs signed off on tech review, so if you're happy, this should be good to merge

**Comment by adellape** [comment]
Looks cool, thanks.

**Pull request merged**